src/Symfony/Bundle/DebugBundle/ExpressionLanguage.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\DebugBundle;
+
+use Symfony\Component\Security\Core\Authorization\ExpressionLanguage as BaseExpressionLanguage;
+
+/**
+ * Adds some functions to the default Symfony Debug ExpressionLanguage.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class ExpressionLanguage extends BaseExpressionLanguage
+{
+    protected function registerFunctions()
+    {
+        parent::registerFunctions();
+        $this->register('strpos', function ($haystack, $needle) {
```

Comment:
Since Symfony 2.6, we introduce a function provider in the ExpressionLanguage component, and it should be used.

See https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Security/Core/Authorization/ExpressionLanguage.php and https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Security/Core/Authorization/ExpressionLanguageProvider.php for the usage in the security component.
The custom ExpressionLanguage is not even needed here; we could just register the provider in the service definition, using a normal ExpressionLanguage instance (the security component must have it for BC)